### PR TITLE
Feature to add i18n - Swedish Language Support

### PR DESCRIPTION
**Pull request by akshitagupta15june**
- **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Feature to add i18n - Swedish Language Support
- **Why was this change needed?** (You can also link to an open issue here)
Fixes issue 🚀 Feature: i18n - #1581 
- **Other information**:
-Kindly review and merge this PR

### CODE REVIEW

**Comment by akshitagupta15june** [comment]
@davidsoderberg @p-fernandez @scopsy please review this pr
previously i gave a norwegian support pr too #1559

File: packages/notification-center/src/i18n/languages/sv.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { ITranslationEntry } from '../lang';
+
+export const SV: ITranslationEntry = {
+  translations: {
+    notifications: 'meddelanden',
```
Comment (davidsoderberg) [inline packages/notification-center/src/i18n/languages/sv.ts:5]: We should use big m here

File: packages/notification-center/src/i18n/languages/sv.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+    markAllAsRead: 'Markera allt som läst',
+    poweredBy: 'poweredBy',
+    settings: 'inställningar',
```
Comment (davidsoderberg) [inline packages/notification-center/src/i18n/languages/sv.ts:8]: We should use `I` here

File: packages/notification-center/src/i18n/languages/sv.ts [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import { ITranslationEntry } from '../lang';
+
+export const SV: ITranslationEntry = {
+  translations: {
+    notifications: 'meddelanden',
+    markAllAsRead: 'Markera allt som läst',
```
Comment (davidsoderberg) [inline packages/notification-center/src/i18n/languages/sv.ts:6]: This should be `Markera alla som lästa`

File: packages/notification-center/src/i18n/languages/sv.ts [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+    poweredBy: 'poweredBy',
```
Comment (davidsoderberg) [inline packages/notification-center/src/i18n/languages/sv.ts:7]: This should be translated with `Använder`

**Review by davidsoderberg — CHANGES_REQUESTED** [review]
The translation have some errors, please correct those...

**Comment by akshitagupta15june** [comment]
Sure i will correct it

**Comment by akshitagupta15june** [comment]
@davidsoderberg please review

**Review by davidsoderberg — APPROVED** [review]

**Comment by akshitagupta15june** [comment]
@davidsoderberg all checks have passed

**Pull request merged**